tests/build/build.go

```diff
@@ -123,6 +126,190 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
+				timeout = time.Minute * 1
+				interval = time.Second * 1
+
+				Eventually(func() bool {
+					pipelineRun, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, "")
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun has not been created yet")
+						return false
+					}
+					return pipelineRun.HasStarted()
+				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start")
+
+				Expect(f.AsKubeAdmin.TektonController.DeleteAllPipelineRunsInASpecificNamespace(testNamespace)).To(Succeed())
+			})
+
+			It("simple build can be triggered manually", func() {
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				component.Annotations[build.BuildRequestAnnotationName] = build.BuildRequestTriggerSimpleBuildAnnotationValue
+				Expect(f.AsKubeAdmin.CommonController.KubeRest().Update(context.TODO(), component)).To(Succeed())
+
+				Eventually(func() bool {
+					pipelineRun, err := f.AsKubeAdmin.HasController.GetComponentPipelineRun(annotationsComponentName, applicationName, testNamespace, "")
+					if err != nil {
+						GinkgoWriter.Println("PipelineRun has not been created yet")
+						return false
+					}
+					return pipelineRun.HasStarted()
+				}, timeout, interval).Should(BeTrue(), "timed out when waiting for the PipelineRun to start")
+
+				Expect(f.AsKubeAdmin.TektonController.DeleteAllPipelineRunsInASpecificNamespace(testNamespace)).To(Succeed())
+
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+				Expect(component.Annotations[build.BuildRequestAnnotationName]).To(BeEmpty(), "request annotation was not deleted")
+			})
+
+			It("PaC provision is finished succesfully", func() {
+				component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+				Expect(err).ShouldNot(HaveOccurred())
+
+				component.Annotations[build.BuildRequestAnnotationName] = build.BuildRequestConfigurePaCAnnotationValue
+				Expect(f.AsKubeAdmin.CommonController.KubeRest().Update(context.TODO(), component)).To(Succeed())
+
+				Eventually(func() (bool, error) {
+					component, err = f.AsKubeAdmin.HasController.GetHasComponent(annotationsComponentName, testNamespace)
+					if err != nil {
+						GinkgoWriter.Printf("cannot get the component: %v\n", err)
+						return false, err
+					}
+
+					statusBytes := []byte(component.Annotations[build.BuildStatusAnnotationName])
+					buildStatus := &build.BuildStatus{}
+
+					err := json.Unmarshal(statusBytes, buildStatus)
+					if err != nil {
+						GinkgoWriter.Printf("cannot unmarshal build status: %v\n", err)
+						return false, err
+					}
+
+					return buildStatus.PaC != nil && buildStatus.PaC.State == "enabled", nil
```

Comment:
Will it be good to check the `pac.merge-url` is not empty and `pac.error-id` value is empty as well?

Reply:
Sure, that's a good idea